addons/block_code/simple_nodes/simple_character/simple_character.gd

```diff
@@ -31,6 +31,27 @@ const PLAYER_KEYS = {
 	}
 }
 
+const PLAYER_JOYSTICK_BUTTONS = {
+	"player_1": {
+		0: {
+			"up": JOY_BUTTON_DPAD_UP,
```

Comment:
Please also map the left stick of each controller to these actions. From my experience play-testing with younger children, they expect the joystick to work and don't use the d-pad.

Reply:
added left joystick controls in 8be0a70